NGitLab/Impl/VariableClient.cs

```diff
@@ -8,6 +9,8 @@ internal abstract class VariableClient
     private readonly string _urlPrefix;
     private readonly API _api;
 
+    private string EnvironmentScopeFilter(string environmentScope = null) => !string.IsNullOrWhiteSpace(environmentScope) ? $"?filter[environment_scope]={Uri.EscapeDataString(environmentScope!)}" : string.Empty;
```

Comment:
```suggestion
    private string EnvironmentScopeFilter(string environmentScope = null) => !string.IsNullOrWhiteSpace(environmentScope) ? $"?filter[environment_scope]={Uri.EscapeDataString(environmentScope)}" : string.Empty;
```

Normally the compiler should be smart enough to not report it as nullable here (due to the use of IsNullOrWhiteSpace). Was there a warning on it?

Reply:
@Toa741 You're completely right.